backend/src/main/java/com/zeros/backend/repositories/RestaurantAffiliateRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.zeros.backend.repositories;
+
+
+import com.zeros.backend.models.RestaurantAffiliate;
+
+public interface RestaurantAffiliateRepository extends Repository<RestaurantAffiliate>{
```

Comment:
Can we extend here [CrudRepository](https://docs.spring.io/spring-data/data-commons/docs/current/api/org/springframework/data/repository/CrudRepository.html) instead ? If yes, there will be no need to implement findByRestaurantId in InMemoryRestaurantAffiliateRepository

Reply:
I think we shoud implicitements the find... because this is just mock implémentation spring dsta jpa mechanisme is not enabled